### DISCUSSION TITLE
On Windows' msi has met an error

### DISCUSSION THREAD

**Discussion by abdulbadii** [neovide/neovide#2656] [General]
Stumbled in installing by use of msi (MS Installer) with message sort of:
Installer encountered error when installing with error code: 2503

Please help how to solve. Thanks before

**Comment by fredizzimo** [comment]
I don't think there's any problems with the msi package we provide. But a quick internet search suggests that the problem is either that the installer fails to run as administrator or that there's something wrong with the file permissions of your temp folder.

You can try manually running the msi as administrator. For the temp file permission issue I don't want to give any advice, but you can find it by searching for the error code.

You can also use one of the alternative ways to install, like scoop.
